packages/transactions/src/clarity/types/intCV.ts

```diff
@@ -1,10 +1,10 @@
-import { IntegerType, intToBigInt } from '@stacks/common';
+import { IntegerType, intToBigInt, bigIntPow } from '@stacks/common';
 import { ClarityType } from '../clarityValue';
 
-const MAX_U128 = BigInt(2) ** BigInt(128) - BigInt(1);
+const MAX_U128 = bigIntPow(BigInt(2), BigInt(128)) - BigInt(1);
```

Comment:
Clever solution! But, because these are constants (module inlines), it would better to define them with as little overhead as possible. These were only math expressions in the first place for clarity to the reader, which could just be comments. So something like:
```ts
const MAX_U128 = BigInt('0xffffffffffffffffffffffffffffffff'); // (2 ** 128 - 1)
const MIN_U128 = BigInt(0);
const MAX_I128 = BigInt('0x7fffffffffffffffffffffffffffffff'); // (2 ** 127 - 1)
// no signed (negative) hex support in bigint constructor
const MIN_I128 = BigInt('-170141183460469231731687303715884105728'); // (-2 ** 127)
```

Reply:
Thanks